src/processPattern.js

```diff
@@ -73,6 +73,19 @@ export default function processPattern(globalRef, pattern) {
                 }
             }
 
+            // Run ignore functions
+            for (let i = 0; i < pattern._ignoreFunctions.length; i++) {
+                debug(`running ignore function against ${file.relativeFrom}`);
+                if (
+                    pattern._ignoreFunctions[i]({
+                        absolutePath: file.absoluteFrom,
+                        relativePath: file.relativeFrom
```

Comment:
I think better arguments is `absolutePath` and `context`, it is allow to know `relative` path and also allow get context

Reply:
I've named it absolute and relative path, as that's mostly related to absoluteFrom and relativeFrom.